packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v3.2.0-dev (Sep 4, 2024)
```

Comment:
Why there are two -dev versions?

Reply:
What would be best to put here? I'm not familiar with the PWA Kit versioning convention yet and would love to know and revise!

Reply:
it should be under 3.1.0-dev since it is the version to be released.